kola/tests/misc/files.go

```diff
@@ -85,6 +91,22 @@ func sugidFiles(m platform.Machine, validfiles []string, mode string) error {
 	return nil
 }
 
+func DeadLinks(c cluster.TestCluster) error {
+	m := c.Machines()[0]
+
+	/* Accept return code 1 for link loop warnings.  */
+	output, err := m.SSH(`sudo find -L / /usr -ignore_readdir_race -xdev -type l 2>/dev/null || [ "x$?" = x1 ]`)
```

Comment:
We don't need the ` = x1` bit or the stderr redirect if we use `find ... -xtype l` instead.